### DISCUSSION TITLE
Alpha channel missing

### DISCUSSION THREAD

**Discussion by daxpandhi** [dlemstra/Magick.NET#611] [General]
### Prerequisites
- [x] I have written a descriptive issue title
- [x] I have verified that I am using the latest version of Magick.NET
- [x] I have searched [open](https://github.com/dlemstra/Magick.NET/issues) and [closed](https://github.com/dlemstra/Magick.NET/issues?q=is%3Aissue+is%3Aclosed) issues to ensure it has not already been reported

### Description
Alpha channel is ignored when saving to several formats that support it. When debugging, I can see that the MagickImage has an alpha channel with the data provided. However, the saved file does not show the A channel in Photoshop or other applications.

Maybe I'm not doing something write, but feels like if Alpha is On/Set, and data is being added it should save too. I've tried it on 7.14.x, 7.15.x and 7.16.x to confirm.

### Steps to Reproduce
Here is a test program to create 3 files that should have alpha channels.

```
    internal class Program
    {
        private static void Main(string[] args)
        {
            MagickImage mm = new MagickImage(MagickColors.Black, 1024, 1024)
            {
                ColorType = ColorType.TrueColorAlpha,
            };

            mm.Alpha(AlphaOption.Set); // Have tried all "positive" options

            float[] data = new float[1024 * 1024 * 4];

            Random r = new Random(256);

            for (int i = 0; i < data.Length; i++)
            {
                data[i] = (float)(Quantum.Max * r.NextDouble());
            }

            using (var pix = mm.GetPixelsUnsafe())
            {
                pix.SetPixels(data);
            }

            mm.BitDepth(16);

            mm.Write("D:\\rgba.psd", MagickFormat.Psd);
            mm.Write("D:\\rgba.tif", MagickFormat.Tiff);
            mm.Write("D:\\rgba.png", MagickFormat.Png);

            mm.Dispose();
        }
    }
```

### System Configuration
- Magick.NET version: `Magick.NET-Q16-HDRI-OpenMP-x64 7.16.0.0`
- Environment (Operating system, version and so on): `Win 7 SP1, Win 10 (all versions) .NET Fx 4.8`
- Additional information:

**Comment by dlemstra** [comment]
I am unable to reproduce your issue:

![afbeelding](https://user-images.githubusercontent.com/10426229/79384155-a92e5880-7f66-11ea-8efd-64574b7f5ab3.png)

`64 bit depth = 16 bit per channel * 4` . This means there is an alpha channel. And when I check the files with ImageMagick (maybe biased?) all the files have an alpha channel. Not sure why Photoshop seems to think that these images have no alpha channel.

**Comment by daxpandhi** [comment]
Very strange. I'll investigate more on my end see if I can reproduce it more extensively or if there is something wrong in my workflow that is causing the problem. Will check and report back. Thanks!

**Comment by dlemstra** [comment]
Feel free to reach out when you have more info.

**Comment by daxpandhi** [comment]
![image](https://user-images.githubusercontent.com/6795809/79385102-ad1da300-7f85-11ea-85be-1e5e527397c0.png)

This is what the untouched output from the code I wrote above gives me, btw.

**Comment by dlemstra** [comment]
32 seems like an 8 bit image? `4 * 8 = 32` and `32 / 3 = 10.6666666667` that seems like a strange bit depth.

**Reply by daxpandhi** [reply]
I called `BitDepth(16)` so not sure why it would be 32. I'll test entire process again on my main computer tomorrow and report back.

**Reply by dlemstra** [reply]
`BitDepth(16)` means 16 bits per channel. And I know PNG can move it down to 8 when it thinks it can.
